File: 백준/Bronze/9498. 시험 성적/시험 성적.js
let fs = require("fs");
let input = fs.readFileSync('/dev/stdin').toString().split("\n");

const score = input[0];

function A(score) {
  if (score >= 90) console.log("A");
  else if (score >= 80) console.log("B");
  else if (score >= 70) console.log("C");
  else if (score >= 60) console.log("D");
  else console.log("F");
}

A(score);

File: 백준/Bronze/9498. 시험 성적/시험 성적.py
a=int(input())
if a>=90 and a<=100:
    score='A'
elif a>=80 and a<=89:
    score='B'
elif a>=70 and a<=79:
    score='C'
elif a>=60 and a<=69:
    score='D'
else:
    score='F'
print(score)